test/component_test.go

```
@@ -0,0 +1,218 @@
+	deployment, err := clientset.AppsV1().Deployments(controllerNamespace).Get(context.Background(), "aws-load-balancer-controller", metav1.GetOptions{})
+	assert.NoError(s.T(), err)
+	assert.NotNil(s.T(), deployment)
+	assert.Equal(s.T(), deployment.Name, "aws-load-balancer-controller")
+	assert.Equal(s.T(), deployment.Namespace, controllerNamespace)
+	assert.Equal(s.T(), *deployment.Spec.Replicas, int32(2))
+
+
+	ingressNamespace := fmt.Sprintf("example-%s", randomID)
+	ingressName := fmt.Sprintf("example-ingress-%s", randomID)
+
+	defer clientset.CoreV1().Namespaces().Delete(context.Background(), ingressNamespace, metav1.DeleteOptions{})
```

golangci-lint (1.64.8)

Error return value of `(k8s.io/client-go/kubernetes/typed/core/v1.NamespaceInterface).Delete` is not checked

(errcheck)